src/sql/workbench/contrib/profiler/browser/profilerEditor.ts

```diff
@@ -464,7 +464,7 @@ export class ProfilerEditor extends EditorPane {
 		let savedViewState = this._savedTableViewStates.get(input);
 
 		this._profilerEditorContextKey.set(true);
-		if (input instanceof ProfilerInput && input.matches(this.input)) {
+		if (input instanceof ProfilerInput && this.input !== undefined && input.matches(this.input)) {
```

Comment:
Why is the input undefined? That shouldn't be happening in the first place I'd think

Reply:
Oh nevermind, it's looking at the previous input. 

Why are we checking if the inputs match anyways? If we have a saved state for the new input shouldn't we always be restoring that?

Reply:
I'm not sure why, but it appears that the setInput function is called by doOpenEditor (a vscode call), and this function overrides the default setInput function of the editorpane instead of the base one, meaning that at the start, the this._input value is not set at all. I'll see if this was the same for the earlier builds or not.